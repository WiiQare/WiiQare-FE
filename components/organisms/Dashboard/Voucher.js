import { CiCircleInfo } from "react-icons/ci";
import { HiUserGroup } from "react-icons/hi";
import ButtonEarn from "../../atoms/Button/Earn";
import WithQR from "../../atoms/Card/Debit/withQR";
import { TabPanel } from "../../atoms/Modal/content";
import { HiCreditCard } from "react-icons/hi2";
import ButtonBuy from "../../atoms/Button/Buy";

const VoucherHome = () => {
	return (
		<div className="p-2 md:pb-8 md:px-6 flex flex-col md:flex-row gap-8 md:gap-12 justify-center">
			{/* <div className="md:min-w-5/6 flex">
				<WithQR />
			</div> */}

			<div className="bg-white drop-shadow-sm rounded-xl py-6 px-24 flex flex-col justify-center items-center gap-8 w-full relative">
				<HiCreditCard size={60} className="text-orange" />
				<div className="space-y-3 text-center w-full flex flex-col justify-center items-center">
					<h1 className="text-gray-800 text-lg">Prenez soin de vos proches </h1>
					<div className="buy">
						<ButtonBuy href={`/voucher/buy`} title={"Envoyer un pass santÃ©"} className={"text-white py-3 px-4"}/>
					</div>
					<span data-popover-target="popover-description" className="absolute top-1 right-6 cursor-pointer text-gray-700 md:text-gray-400"><CiCircleInfo size={25} /></span>

					<div data-popover id="popover-description" role="tooltip" class="absolute z-50 invisible inline-block text-sm font-light text-gray-500 transition-opacity duration-300 bg-white border border-gray-200 rounded-lg shadow-sm opacity-0 w-72 ">
						<div class="p-3 space-y-2 text-left">
							<h3 class="font-semibold text-gray-900 dark:text-white">Description</h3>
							<p>
								Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit cumque hic sunt nemo fugiat porro, exercitationem commodi velit dolor eos nam sint sapiente iure maxime vitae nisi error autem tempora.
							</p>

							<a href="#" class="flex items-center font-medium text-blue-600 dark:text-blue-500 dark:hover:text-blue-600 hover:text-blue-700">Read more <svg class="w-4 h-4 ml-1" aria-hidden="true" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z" clip-rule="evenodd"></path></svg></a>
						</div>
						<div data-popper-arrow></div>
					</div>
				</div>
			</div>

			<div className="bg-white drop-shadow-sm rounded-xl py-6 px-24 flex flex-col justify-center items-center gap-8 w-full relative">
				<HiUserGroup size={60} className="text-sky" />
				<div className="space-y-3 text-center w-full flex flex-col items-center">
					<h1 className="text-gray-800 text-lg">Parrainez vos amis</h1>
					<ButtonEarn modal={{ title: "Invite your friends" }}></ButtonEarn>
					<span data-popover-target="popover-description" className="absolute top-1 right-6 cursor-pointer text-gray-700 md:text-gray-400"><CiCircleInfo size={25} /></span>

					<div data-popover id="popover-description" role="tooltip" class="absolute z-50 invisible inline-block text-sm font-light text-gray-500 transition-opacity duration-300 bg-white border border-gray-200 rounded-lg shadow-sm opacity-0 w-72 ">
						<div class="p-3 space-y-2 text-left">
							<h3 class="font-semibold text-gray-900 dark:text-white">Description</h3>
							<p>
								Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit cumque hic sunt nemo fugiat porro, exercitationem commodi velit dolor eos nam sint sapiente iure maxime vitae nisi error autem tempora.
							</p>

							<a href="#" class="flex items-center font-medium text-blue-600 dark:text-blue-500 dark:hover:text-blue-600 hover:text-blue-700">Read more <svg class="w-4 h-4 ml-1" aria-hidden="true" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z" clip-rule="evenodd"></path></svg></a>
						</div>
						<div data-popper-arrow></div>
					</div>
				</div>
			</div>
		</div>
	);
};

export default VoucherHome;
