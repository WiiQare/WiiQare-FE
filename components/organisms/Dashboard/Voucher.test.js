import { render } from '@testing-library/react';
import Voucher from './Voucher';

describe('Voucher', () => {
  it('renders', () => {
    const { container } = render(<Voucher />);
    expect(container).toMatchSnapshot();
  });
});
