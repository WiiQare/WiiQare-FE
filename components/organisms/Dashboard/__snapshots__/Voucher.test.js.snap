// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Voucher renders 1`] = `
<div>
  <div
    class="p-2 md:pb-8 md:px-6 flex flex-col md:flex-row gap-8 md:gap-12 justify-center"
  >
    <div
      class="bg-white drop-shadow-sm rounded-xl py-6 px-10 md:px-24 flex flex-col justify-center items-center gap-8 w-full relative"
    >
      <svg
        aria-hidden="true"
        class="text-orange"
        fill="currentColor"
        height="60"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 24 24"
        width="60"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M4.5 3.75a3 3 0 00-3 3v.75h21v-.75a3 3 0 00-3-3h-15z"
        />
        <path
          clip-rule="evenodd"
          d="M22.5 9.75h-21v7.5a3 3 0 003 3h15a3 3 0 003-3v-7.5zm-18 3.75a.75.75 0 01.75-.75h6a.75.75 0 010 1.5h-6a.75.75 0 01-.75-.75zm.75 2.25a.75.75 0 000 1.5h3a.75.75 0 000-1.5h-3z"
          fill-rule="evenodd"
        />
      </svg>
      <div
        class="space-y-3 text-center w-full flex flex-col justify-center items-center z-40"
      >
        <h1
          class="text-gray-800 text-lg"
        >
          Prenez soin de vos proches 
        </h1>
        <div
          class="buy"
        >
          <a
            class="flex items-center gap-2 justify-between rounded-lg bg-orange py-2 px-3 effect-up shadow text-white py-3 px-4"
            href="/voucher/buy"
          >
            <span
              class="bg-white text-sky p-1 rounded-md"
            >
              <svg
                aria-hidden="true"
                fill="none"
                height="1em"
                stroke="currentColor"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M12 4v16m8-8H4"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
            </span>
            <span
              class="text-sm font-light"
            >
              Envoyer un pass santé
            </span>
          </a>
        </div>
        <span
          class="absolute z-50 top-1 right-6 cursor-pointer text-gray-700 md:text-gray-400"
          data-popover-target="popover-description"
        >
          <svg
            fill="currentColor"
            height="25"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="25"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              id="Circle_Info"
            >
              <g>
                <g>
                  <path
                    d="M11.5,15a.5.5,0,0,0,1,0h0V10.981a.5.5,0,0,0-1,0Z"
                  />
                  <circle
                    cx="12"
                    cy="8.999"
                    r="0.5"
                  />
                </g>
                <path
                  d="M12,2.065A9.934,9.934,0,1,1,2.066,12,9.945,9.945,0,0,1,12,2.065Zm0,18.867A8.934,8.934,0,1,0,3.066,12,8.944,8.944,0,0,0,12,20.932Z"
                />
              </g>
            </g>
          </svg>
        </span>
        <div
          class="absolute z-50 invisible inline-block text-sm font-light text-gray-500 transition-opacity duration-300 bg-white border border-gray-200 rounded-lg shadow-sm opacity-0 w-72 "
          data-popover="true"
          id="popover-description"
          role="tooltip"
        >
          <div
            class="p-3 space-y-2 text-left"
          >
            <h3
              class="font-semibold text-gray-900 dark:text-white"
            >
              Description
            </h3>
            <p>
              <strong>
                La pass santé WiiQare  
              </strong>
               est un moyen rapide et sans frais de faciliter l’accès aux soins de qualité pour vos proches en Afrique.
            </p>
            <a
              class="flex items-center font-medium text-blue-600 dark:text-blue-500 dark:hover:text-blue-600 hover:text-blue-700"
              href="www.wiiqare.com"
            >
              Read more 
              <svg
                aria-hidden="true"
                class="w-4 h-4 ml-1"
                fill="currentColor"
                viewBox="0 0 20 20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
                  fill-rule="evenodd"
                />
              </svg>
            </a>
          </div>
          <div
            data-popper-arrow="true"
          />
        </div>
      </div>
    </div>
    <div
      class="bg-white drop-shadow-sm rounded-xl py-6 px-10 md:px-24 flex flex-col justify-center items-center gap-8 w-full relative"
    >
      <svg
        aria-hidden="true"
        class="text-sky"
        fill="currentColor"
        height="60"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 20 20"
        width="60"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M13 6a3 3 0 11-6 0 3 3 0 016 0zM18 8a2 2 0 11-4 0 2 2 0 014 0zM14 15a4 4 0 00-8 0v3h8v-3zM6 8a2 2 0 11-4 0 2 2 0 014 0zM16 18v-3a5.972 5.972 0 00-.75-2.906A3.005 3.005 0 0119 15v3h-3zM4.75 12.094A5.973 5.973 0 004 15v3H1v-3a3 3 0 013.75-2.906z"
        />
      </svg>
      <div
        class="space-y-3 text-center w-full flex flex-col items-center z-40"
      >
        <h1
          class="text-gray-800 text-lg"
        >
          Invitez vos amis
        </h1>
        <button
          class="flex bg-sky px-4 py-4 justify-center text-white items-center rounded-xl hover:shadow-md gap-2 w-auto md:w-52"
        >
          <svg
            fill="currentColor"
            height="25"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="25"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              id="Dollar"
            >
              <g>
                <path
                  d="M12,21.934A9.934,9.934,0,1,1,21.934,12,9.945,9.945,0,0,1,12,21.934ZM12,3.066A8.934,8.934,0,1,0,20.934,12,8.944,8.944,0,0,0,12,3.066Z"
                />
                <path
                  d="M14.5,13.5a2.006,2.006,0,0,1-2,2v1.01a.5.5,0,0,1-1,0V15.5H10.25a.5.5,0,0,1,0-1H12.5a1,1,0,0,0,0-2h-1a2,2,0,0,1,0-4V7.49a.5.5,0,0,1,1,0V8.5h1.25a.5.5,0,0,1,0,1H11.5a1,1,0,0,0,0,2h1A2.006,2.006,0,0,1,14.5,13.5Z"
                />
              </g>
            </g>
          </svg>
          <span
            class="text-xs font-light"
          >
            Gagnez 5$
          </span>
        </button>
        <span
          class="absolute top-1 right-6 cursor-pointer text-gray-700 md:text-gray-400"
          data-popover-target="popover-description"
        >
          <svg
            fill="currentColor"
            height="25"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="25"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              id="Circle_Info"
            >
              <g>
                <g>
                  <path
                    d="M11.5,15a.5.5,0,0,0,1,0h0V10.981a.5.5,0,0,0-1,0Z"
                  />
                  <circle
                    cx="12"
                    cy="8.999"
                    r="0.5"
                  />
                </g>
                <path
                  d="M12,2.065A9.934,9.934,0,1,1,2.066,12,9.945,9.945,0,0,1,12,2.065Zm0,18.867A8.934,8.934,0,1,0,3.066,12,8.944,8.944,0,0,0,12,20.932Z"
                />
              </g>
            </g>
          </svg>
        </span>
        <div
          class="absolute z-50 invisible inline-block text-sm font-light text-gray-500 transition-opacity duration-300 bg-white border border-gray-200 rounded-lg shadow-sm opacity-0 w-72 "
          data-popover="true"
          id="popover-description"
          role="tooltip"
        >
          <div
            class="p-3 space-y-2 text-left"
          >
            <h3
              class="font-semibold text-gray-900 dark:text-white"
            >
              Description
            </h3>
            <p>
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit cumque hic sunt nemo fugiat porro, exercitationem commodi velit dolor eos nam sint sapiente iure maxime vitae nisi error autem tempora.
            </p>
            <a
              class="flex items-center font-medium text-blue-600 dark:text-blue-500 dark:hover:text-blue-600 hover:text-blue-700"
              href="#"
            >
              Read more 
              <svg
                aria-hidden="true"
                class="w-4 h-4 ml-1"
                fill="currentColor"
                viewBox="0 0 20 20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
                  fill-rule="evenodd"
                />
              </svg>
            </a>
          </div>
          <div
            data-popper-arrow="true"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
